README.md

```diff
@@ -1,2 +1,20 @@
-# trial-viewer
-A pure vanilla json viewer for trials
+# TrialViewer
+TrialViewer is a vanilla JavaScript tool for visualising trial JSON data. It is inspired from [JSONViewer].
```

Comment:
Did you change the JSONViewer library?

Reply:
No. I just use it directly.

Reply:
Then it is not inspired. We just created a repo to demo how to use this lib.